tox.ini

```diff
@@ -1,60 +1,35 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37,pypy,
-    py27-pyramid{12,13,14,15,16,17,18,19},
+    py34,py35,py36,py37,py38,py39,pypy3,
+    py39-pyramid{110,20}
     docs,
-    {py2,py3}-cover,coverage
+    coverage
+
+deps =
```

Comment:
These lines are in the wrong place, they need to be in the `[testenv]` section of the `tox.ini`. They won't work if they are located here, as seen in the output from Travis:

```
py39-pyramid110 installed: hupper==1.10.2,PasteDeploy==2.1.1,plaster==1.0,plaster-pastedeploy==0.7,pyramid==2.0,pyramid-mailer @ file:///home/travis/build/Pylons/pyramid_mailer/.tox/.tmp/package/1/pyramid_mailer-0.15.1.zip,repoze.sendmail==4.4.1,transaction==3.0.1,translationstring==1.4,venusian==3.0.0,WebOb==1.8.7,zope.deprecation==4.4.0,zope.interface==5.4.0
```

Notice that it is installing Pyramid 2.0 not Pyramid 1.10.